app/assets/javascripts/price_policy.js

```diff
@@ -21,6 +21,7 @@ $(document).ready(function() {
     const isDisabled = !$checkbox.prop("checked");
     $cells.toggleClass("disabled", isDisabled);
     $cells.find("input[type=text], input[type=hidden], input[type=checkbox]").not($checkbox).each((_i, elem) => hardToggleField($(elem), isDisabled));
+    $cells.find(".per-minute").toggleClass("hidden", isDisabled);
```

Comment:
why is this needed?  will any other pages be affected?

Reply:
When this wasn't here, rate per minute was visible even when inputs were disabled (and showed no values). 

Staging:
<img width="847" alt="image" src="https://github.com/tablexi/nucore-open/assets/28798610/1d864e2b-7e24-40d2-b7d6-17fd7dccfd70">

Locally:
<img width="877" alt="image" src="https://github.com/tablexi/nucore-open/assets/28798610/ac4c98c1-96c7-4f8a-a57f-3abb5cf57e9a">


I can try a way of only changing this for Stepped billing, but I think it's pointless to show a "X rate per minute" if that group can't purchase and we don't show the rate value anywhere

Reply:
If the behavior is the same for stepped billing and non-stepped billing then let's fix it in both places.  sound good?